lib/perl/Genome/InstrumentData/AlignmentResult/Merged.pm

```diff
@@ -739,4 +740,46 @@ sub estimated_gtmp_for_instrument_data  {
     );
 }
 
+
+sub get_superseding_results {
+    my ($self, $test_name_ok) = @_;
+
+    my @ars  = $self->collect_individual_alignments;
+    my $ars  = Set::Scalar->new(@ars);
+    
+    my %ct;
+    my @founds = ();
+
+    for my $ar (@ars) {
+        my @sup_mrs = $ar->get_merged_alignment_results;
+        for my $sup_mr (@sup_mrs) {
+            if ($ct{$sup_mr->id}) {
+                $ct{$sup_mr->id}++;
+            }
+            else {
+                $ct{$sup_mr->id} = 1;
+            }
+        }
+    }
+
+    for my $supmr_id (keys %ct) {
+        next unless $ct{$supmr_id} == $ars->size;
+        next if $supmr_id eq $self->id;
+
+        my $supmr  = __PACKAGE__->get($supmr_id);
+
+        unless ($test_name_ok) {
+            next if $supmr->test_name;
```

Comment:
Both [`get_merged_alignments`](https://github.com/genome/genome/blob/be14ea7b97b765d273a0268a55b47ff00d789e5b/lib/perl/Genome/InstrumentData/AlignmentResult.pm#L1927) and [`collect_individual_alignments`](https://github.com/genome/genome/blob/be14ea7b97b765d273a0268a55b47ff00d789e5b/lib/perl/Genome/InstrumentData/AlignmentResult/Merged.pm#L359) respect the test name on the result upon which they're called, so this option (when false) is equivalent to returning immediately if `$self` has a `test_name` set.